c3.js

```diff
@@ -546,8 +546,20 @@
         // show/hide if manual culling needed
         if ((withUpdateXDomain || withUpdateXAxis) && targetsToShow.length) {
             if (config.axis_x_tick_culling && tickValues) {
-                for (i = 1; i < tickValues.length; i++) {
-                    if (tickValues.length / i < config.axis_x_tick_culling_max) {
+                var cullSetLength = tickValues.length;
```

Comment:
I believe you want to add this to `/src/zoom.js` not `c3.js` as this is the result of the build.

(same as https://github.com/c3js/c3/pull/1733/files#r69476315)